### DISCUSSION TITLE
Mode 9 unload problem

### DISCUSSION THREAD

**Discussion by xkolive** [Courseplay/courseplay#7456] [Q&A] [answered]
Hi, i have a problem with mode 9. Frontloader unload only 5-10 % from showel and continue course.
Please watch video, when frontloader start 2 course i enable console for log...
https://www.youtube.com/watch?v=6QK2vZUjWh0 

Thanks

**Comment by Tensuko** [comment, marked as answer]
Shovelposition

**Reply by xkolive** [reply]
Which position is correct? If i set shovel higher, loader only move shovel but nothing unload...

**Reply by Tensuko** [reply]
Unloadposition a bit less down.
If to much down it might not unload all. That is a bit strange behavior and hard to explain why. It got to do with the trigger of the shovel and the load area of the trailer.

**Reply by xkolive** [reply]
It is hard to find correct position for working, but now i know how to setup.

Thanks you so much...
